content/docs/started/getting-started-multipass.md

```diff
@@ -7,7 +7,7 @@ weight = 2
 This document outlines the steps that you can take to get your local
 installation of Kubeflow running on top of Microk8s, a single-node Kubernetes cluster. Microk8s requires Linux; if you are not on a Linux system, you can use Multipass to create a Linux VM on your native hypervisor.
 
-By the end of this document, you'll have a local installation of a Kubernetes cluster along with all the default core components of Kubeflow deployed as services in the pods. You can access the Kubeflow dashboard, JupyterHub notebooks, and dashboards of other Kubeflow components.
+By the end of this document, you'll have a local installation of a Kubernetes cluster along with all the default core components of Kubeflow deployed as services in the pods. You can access the Kubeflow dashboard, Jupyter notebooks, and dashboards of other Kubeflow components.
```

Comment:
Same comment as other docs, consider changing:

Jupyter notebooks

to:

Jupyter Notebook servers

Reply:
I'm going to stick with "Jupyter notebooks" for simplicity across the docs, as I think that will be clearer to our readers. It may come across as a little confusing to sometimes use "Notebook" and other times "notebook", though I do take your point about correctness here.